app/main.py

```diff
@@ -42,4 +45,21 @@
     },
     openapi_tags=tags_metadata
 )
+
 app.include_router(router, prefix='')
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins='*',
+    allow_credentials=True,
+    allow_methods=["get", "post", "options"],
```

Comment:
Does this allow anyone to do a POST request?

Reply:
It allows any website to query the API but that just browser security, you could/can always make request direct e.g. `curl -X 'POST' 'http://127.0.0.1:8000/arcnet/classify_cutout/'`